core/main.c

```diff
@@ -68,7 +68,8 @@ void usage() {
 void parse_args(int argc, char *argv[]) {
   int ch;
 
-  while ((ch = getopt(argc, argv, "rti:T:")) != -1) {
+  pod_t *pod = get_pod();
```

Comment:
`parse_args()` gets called before the pod structure is initialized.  Can some fields get added to `struct arguments` to parse the args then those arguments get passed off to the `pod_t` following the `init_pod()` call in `main()`